testing/src/main/java/io/stargate/it/storage/SkipIfBundleNotAvailable.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.it.storage;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Annotates a test class or method to check whether a bundle has been built and deployed i.e. it's
+ * available for testing.
+ *
+ * <p>Uses {@link BundleAvailableCondition}.
+ */
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(BundleAvailableCondition.class)
+public @interface SkipIfBundleNotAvailable {
+
+  /** The name of the bundle to check for. */
+  String bundleName();
+}
```

Comment:
nit: rename to `AssumeBundleAvailable { String name(); } `? 

This should allow usage as `@AssumeBundleAvailable("grpc")`, which, I guess, might be more intuitive in the JUnit context..

Reply:
or perhaps `@IfBundleAvailable("grpc")`